src/solver/conv_hip_implicit_gemm_mlir_cpp_bwd.cpp

```diff
@@ -57,46 +57,8 @@ ConvHipImplicitGemmMlirCppBwd::CalculateGemmSize(const ConvolutionContext& ctx)
 
 bool ConvHipImplicitGemmMlirCppBwd::IsApplicable(const ConvolutionContext& ctx) const
 {
-#if MIOPEN_USE_MLIR
-    if(miopen::IsDisabled(MIOPEN_DEBUG_CONV_HIP_IMPLICIT_GEMM_MLIR_CPP_BWD{}))
```

Comment:
[Informative] Changing `miopen::IsDisabled(...)` to `!miopen::IsEnabled(...)` is the "standard" way for disabling a solver by default while keeping it available.

Reply:
Will do! I agree this is a better way.

Reply:
Ok, I am expecting new commit(s) in this PR.

Reply:
Done